packager/hls/base/master_playlist.cc

```diff
@@ -463,10 +468,12 @@ void AppendPlaylists(const std::string& default_audio_language,
 
 MasterPlaylist::MasterPlaylist(const std::string& file_name,
                                const std::string& default_audio_language,
-                               const std::string& default_text_language)
+                               const std::string& default_text_language,
+                               const bool segment_sap_aligned)
```

Comment:
Remove "const". We don't usually prefix primitive types function parameters with "const".

Can you also rename the parameter to: is_independent_segments?